modelling/src/main/java/org/axonframework/modelling/command/inspection/AnnotatedAggregate.java

```diff
@@ -429,11 +427,6 @@ private Object handle(CommandMessage<?> commandMessage) throws Exception {
                     m -> handler.handle(commandMessage, aggregateRoot)
             ).proceed();
         }
-
-        if (aggregateRoot == null) {
```

Comment:
It's not really related to this PR, actually. I just found out this branch isn't necessary, as the responsibility of returning an identifier in case a creation handler returns an Aggregate instance lies with the AggregateAnnotationCommandHandler.

Reply:
Actually, the code was unreachable. `aggregateRoot` couldn't ever be null, as it would have raised an NPE a few lines above. Also, the `handle` method here handles a command on an existing instance. The `registerRoot` method is used to initialize a new aggregate root in this instance.

Reply:
Thanks for clarifying :+1: